src/main/java/seedu/address/model/person/PredicateContainer.java

```diff
@@ -0,0 +1,184 @@
+package seedu.address.model.person;
+
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Contains all predicates for FindCommand
+ */
+public class PredicateContainer {
+    private NameContainsKeywordsPredicate nameContainsKeywordsPredicate;
+    private PhoneContainsKeywordsPredicate phoneContainsKeywordsPredicate;
+    private EmailContainsKeywordsPredicate emailContainsKeywordsPredicate;
+    private DepartmentContainsKeywordsPredicate departmentContainsKeywordsPredicate;
+    private RoleContainsKeywordsPredicate roleContainsKeywordsPredicate;
+
+    public PredicateContainer() {
```

Comment:
Perhaps there should be an explanation/JavaDoc why the constructor is empty?

Reply:
Will remove constructor